ebodac/src/main/resources/webapp/js/controllers.js

```diff
@@ -542,17 +546,21 @@
             $scope.lookupRefresh = !$scope.lookupRefresh;
         };
 
+        $scope.refreshGridAndStayOnSamePage = function() {
```

Comment:
on enrollment advanced you always have only one page, so this changes are not needed